wpt/tests/browsing_context/command_navigate.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_browsing_context_navigate_wait_none(bidi_session, wait_for_event,
```

Comment:
How about testing "interactive" and "complete". The event order should be different. Perhaps the test can be parameterized by wait condition and resulting event order, although that'll make rather abstract. Sometimes just copying test code is better, not sure in this case.

Reply:
Out of scope of this PR.